app/static/src/app/store/language/state.ts

```diff
@@ -0,0 +1,7 @@
+import { Language } from "../translations/locales";
```

Comment:
The idea is to put the directive and components into an npm package isn't it? In that case, shouldn't store types which the components and directive will need to access be defined in the package? So that would be Language state, language actions (for the drop down switch) and I guess the top level store state which has language as a property - something like TranslatableState in HINT (though that has lang-related props at top level, not in a 'sub-state'). Or are you intending to do the first implementation of the components within wodin then move out to a package?